### PR TITLE
Add Dockerfile, templates, and documentation to use the Dockerfile

### PR DESCRIPTION
**Pull request by syncrou**
Add a Dockerfile including default `script.rb` and `dug_rules.yml` templates.

Added Docs to `README.md` explaining how best to set up the container for continuous DUG goodness!

### CODE REVIEW

**Comment by chrisarcand** [comment]
I haven't forgotten about this, just needed more time to finally acquaint myself with Docker and will have more specific feedback soon <3

Some initial thoughts though:
- I wish the number of specific copy/pastey steps were reduced, if we could somehow put the Google token in the image build itself that would be ideal (instead of literally shelling in to do it)
- As the templates are checked in to git, changing them to your liking adds changes to your git staging. Would prefer some way that these are ignored, changed after the fact in the image, something that doesn't dirty that.

Maybe a build script or something would fix these issues. <- Know I don't know what I'm talking about here as it relates to Dockerfiles, etc, hence my research needed :)

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM ruby:2.3.1
+MAINTAINER Drew Bomhof (syncrou) https://github.com/syncrou
```
Comment (chrisarcand) [inline Dockerfile:2]: [`MAINTAINER` is deprecated soon; use `LABEL` instead](https://github.com/docker/docker/blob/master/docs/deprecated.md#maintainer-in-dockerfile)

**Comment by syncrou** [comment]
@chrisarcand - Thanks so much for the feedback!  I'll dig into your requests.

**Comment by chrisarcand** [comment]
I am closing this given the following: https://chrisarcand.com/retiring-dug/

Please feel free to continue on with your own fork if you so desire and thanks so much for contributing to this project!!!! ❤️ ❤️ ❤️

**Pull request closed without merging**